README.md

```diff
@@ -22,6 +22,11 @@ Run the tests (servers need not be running, and will be shut down if they are):
 make test
 ```
 
+Note that `docker-compose`, and therefore `make test`, will only build a Docker image for the
```

Comment:
`docker-compose build` will rebuild an image, and `docker-compose build --no-cache` will rebuild without the cache. `make reset` is in there for this kind of use case. It might be helpful to note these things as well

Reply:
updated